test/flutter_circular_chart_test.dart

```diff
@@ -1,7 +1 @@
-import 'package:test/test.dart';
-
-import 'package:flutter_circular_chart/flutter_circular_chart.dart';
-
-void main() {
-
-}
+void main() {}
```

Comment:
🙄 oh boy haha no tests! 

too bad. not sure i even know of a great way to test this sorta thing that isn't just super impl detail aware.

Reply:
Yeah, there's a lot of code cleanup as well but I figured that can be its own PR. I'll think about tests.